packages/react-components/src/components/SendBoxErrorBar.tsx

```diff
@@ -11,31 +11,33 @@ export interface SendBoxErrorBarProps {
   /** Error message to render */
   message?: string;
   /**
-   * Timeout in ms after which the error bar disappears.
+   * Timeout delay in ms after which the error bar disappears.
    * If undefined, the error bar will never disappear.
    */
-  timeout?: number;
+  timeoutDelay?: number;
```

Comment:
nit. 

I am curious can we just call this delay?

timeoutDelay sounds redundant however I dont have a strong opinion.

Reply:
`SendBoxError.delay`
Just by looking at the property in association with the component name, I can't understand what it does.
`timeoutDelay` is also not much better in this case.
How about `hideAfterMs`?